src/main/java/com/ldtteam/structurize/items/ModItems.java

```diff
@@ -53,5 +70,8 @@ private ModItems() { /* prevent construction */ }
         scanTool  = register("sceptersteel", () -> new ItemScanTool(ModItemGroups.STRUCTURIZE));
         tagTool   = register("sceptertag", () -> new ItemTagTool(ModItemGroups.STRUCTURIZE));
         caliper   = register("caliper", () -> new ItemCaliper(properties));
+
+        vanillaCaveAir = registerVanillaOverride(Blocks.CAVE_AIR, () -> new AirItem(Blocks.CAVE_AIR, new Item.Properties()));
```

Comment:
There has to be a better way todo this!

Overriding vanilla items can not be the way here?!?!?

Based on the issues I have seen and the changes mentioned in this pr, such a compatibility breaking change requires much much much much more explanation, documentation and research into why no other option can be chosen and as to why it is needed!

Reply:
I'd probably just make a normal air item for the usage in the guy, and give it in nbt the specific air value

Reply:
Like I commented above, I think it should be in our namespace to avoid compat issues. But being in our namespace I don't see a problem. Either you have three items, or you got 1 item with 3 nbt states.

Reply:
The namespace does not actually mean much, it was just my decision that blocks that are already in vanilla space should have items also in vanilla space, I will put it into our namespace as you wish

Reply:
Then i agree.